README.md

```diff
@@ -42,55 +37,48 @@ fits for your app.
+**Not sure about which APIs are right for your app?** Read our [helpful blog
+post](https://medium.com/slack-developer-blog/getting-started-with-slacks-apis-f930c73fc889) that explains and compares
+the options. If you're still not sure, [reach out for help](#getting-help) and our community can guide you.
 
 ## Examples
 
 ### Posting a message with Web API
 
-Your app will interact with the Web API through the `WebClient` object, which a top level export
-from this package. At a minimum, you need to instantiate it with a token. The example below shows
-how to post a message into a channel, DM, MPDM, or group. This will require either the
-`chat:user:write` or `chat:bot:write` scopes.
+Your app will interact with the Web API through the `WebClient` object, which a top level export from this package. You
+need to instantiate it with a token. The example below shows how to post a message into a channel, DM, MPDM, or group.
+This will require either the `bot`, `chat:user:write` or `chat:bot:write` scopes.
 
 ```javascript
 const { WebClient } = require('@slack/client');
 
-// An access token (from your Slack app or custom integration - xoxa, xoxp, or xoxb)
+// An access token (from your Slack app or custom integration - xoxp, xoxb)
 const token = process.env.SLACK_TOKEN;
 
 const web = new WebClient(token);
 
 // This argument can be a channel ID, a DM ID, a MPDM ID, or a group ID
 const conversationId = 'C1232456';
 
-// See: https://api.slack.com/methods/chat.postMessage
-web.chat.postMessage({ channel: conversationId, text: 'Hello there' })
-  .then((res) => {
-    // `res` contains information about the posted message
-    console.log('Message sent: ', res.ts);
-  })
-  .catch(console.error);
+(async () => {
+  // See: https://api.slack.com/methods/chat.postMessage
+  const res = await web.chat.postMessage({ channel: conversationId, text: 'Hello there' });
+
+  // `res` contains information about the posted message
+  console.log('Message sent: ', res.ts);
+})();
 ```
 
-The `WebClient` object makes it simple to call any of the
-[**over 130 Web API methods**](https://api.slack.com/methods). See the
-[guide](http://slackapi.github.io/node-slack-sdk/web_api) for details.
+The `WebClient` object makes it simple to call any of the [**over 130 Web API methods**](https://api.slack.com/methods).
+See the [guide](http://slackapi.github.io/node-slack-sdk/web_api) for details.
 
-### Posting a message with the Real-Time Messaging API
+### Using the Real-Time Messaging API
 
-Your app will interact with the RTM API through the `RTMClient` object, which is a top level
-export from this package. At a minimum, you need to instantiate it with a token, usually a
-bot token.
+Your app will interact with the RTM API through the `RTMClient` object, which is another top level export from this
```

Comment:
i have mixed feelings about adding any friction to the README. tbh, i was thinking of removing the whole "Requirements" section, because i think that's sort of a detail that better positioned after you understand what the package even does.

there is a one-liner here: https://github.com/slackapi/node-slack-sdk/pull/702/files#diff-7c888464716dcf5504f56ea45fdeb29cR35

do you feel strongly that we need to discourage RTM use? i don't actually think that's helpful for people who are just getting off the ground. i'd rather them use something, anything - and then we can talk to them about how to choose the most appropriate technology for their needs (and sometimes this will be RTM!).